Fixes build with Python >= 3.6:

------
Making all in libgnomeui
make[2]: Entering directory '/tmp/guix-build-libgnomeui-2.24.5.drv-0/libgnomeui-2.24.5/libgnomeui'
  GEN    stamp-gnome-marshal.h
  GEN    stamp-gnometypebuiltins.h
INFO: Reading ./gnome-marshal.list...
  GEN    gnome-marshal.h
Traceback (most recent call last):
  File "/gnu/store/...-glib-2.54.2-bin/bin/glib-mkenums", line 688, in <module>
    process_file(fname)
  File "/gnu/store/...-glib-2.54.2-bin/bin/glib-mkenums", line 420, in process_file
    line = curfile.readline()
  File "/gnu/store/...-python-3.6.3/lib/python3.6/codecs.py", line 321, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xf1 in position 1009: invalid continuation byte
make[2]: *** [Makefile:1109: stamp-gnometypebuiltins.h] Error 1
make[2]: Leaving directory '/tmp/guix-build-libgnomeui-2.24.5.drv-0/libgnomeui-2.24.5/libgnomeui'
make[1]: *** [Makefile:369: all-recursive] Error 1
make[1]: Leaving directory '/tmp/guix-build-libgnomeui-2.24.5.drv-0/libgnomeui-2.24.5'
make: *** [Makefile:296: all] Error 2
------

Patch copied from upstream source repository:

https://git.gnome.org/browse/libgnomeui/commit/?id=30334c28794ef85d8973f4ed0779b5ceed6594f2

From 30334c28794ef85d8973f4ed0779b5ceed6594f2 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Mon, 7 Aug 2017 13:15:26 -0400
Subject: [PATCH] gnome-scores.h: Convert to UTF-8

The new Python `glib-mkenums` barfs on ISO-8859-1 input.
---
 libgnomeui/gnome-scores.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libgnomeui/gnome-scores.h b/libgnomeui/gnome-scores.h
index 911b12d1..4e641d22 100644
--- a/libgnomeui/gnome-scores.h
+++ b/libgnomeui/gnome-scores.h
@@ -27,7 +27,7 @@
  * "High Scores" Widget 
  *
  * AUTHOR: 
- * Horacio J. Peña <horape@compendium.com.ar>
+ * Horacio J. PeÃ±a <horape@compendium.com.ar>
  *
  * This is free software (under the terms of the GNU LGPL)
  *
